mcp/src/minecraft/com/elementfx/tvp/ad/client/gui/inventory/GuiRohirrimCrafting.java

```diff
@@ -0,0 +1,49 @@
+package com.elementfx.tvp.ad.client.gui.inventory;
+
+import com.elementfx.tvp.ad.inventory.ContainerElvenWorkbench;
+import com.elementfx.tvp.ad.inventory.ContainerRohirrimWorkbench;
+
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.inventory.ContainerWorkbench;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+public class GuiRohirrimCrafting extends GuiContainer
+{
+	private static final ResourceLocation ROHIRRIM_CRAFTING_TABLE_GUI_TEXTURES = new ResourceLocation("textures/gui/container/crafting_table.png");
```

Comment:
No other diferences for sure. It should be alright with the constructors, I just wasnt sure with all the classes, and didnt want to connect them really together with constructors etc before you approve the system) I'll do it. I'll try my best with other classes as well, I wasnt too sure how to handle Crafting managers for instance, but your idea further below seems nice!